znai-client/src/main/java/org/testingisdocumenting/znai/client/upload/DocUploader.java

```diff
@@ -0,0 +1,88 @@
+package org.testingisdocumenting.znai.client.upload;
+
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.http.StatusLine;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.entity.ContentType;
+import org.apache.http.entity.FileEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.testingisdocumenting.znai.client.ZipTask;
+import org.testingisdocumenting.znai.console.ConsoleOutputs;
+import org.testingisdocumenting.znai.console.ansi.Color;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class DocUploader {
+    private static final String USER = System.getProperty("USER", "");
```

Comment:
that is a good version one. I think in the next few versions we should move into using `Principal` on a server side. 
Assuming that principla is going to be set (e.g. nginx in front)